### PR TITLE
[PPP-3365] fixed bug of plugin loading interaction with PUC

### PR DESCRIPTION
**Pull request by andrey-markelov**
@pamval @rmansoor please review and merge
and then this related PR: https://github.com/pentaho/pentaho-analyzer/pull/852

### CODE REVIEW

**Comment by pamval** [comment]
@rfellows can you please review ?

File: user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java [hunk 1]
```diff
@@ -359,6 +372,34 @@ public void setCurrentTabSaveEnabled( boolean enabled ) {
     }
   }
 
+  public void setTabSaveEnabled( String frameId, boolean enabled ) {
+    PentahoTab tab = getTabByFrameId( frameId );
+    if( tab != null ) {
```
Comment (rfellows) [inline user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java]: checkstyle violation. there are probably others. please run the checkstyle ant targets and fix any issues in the files you are touching.

**Comment by andrey-markelov** [comment]
Updated

**Comment by pamval** [comment]
@rfellows please review again

**Pull request merged**